backend/src/main/java/de/jannisaziz/backend/user/UserController.java

```diff
@@ -0,0 +1,42 @@
+package de.jannisaziz.backend.user;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/api/users")
+public class UserController {
+
+    private final UserService service;
+
+    @GetMapping("/id/{userId}")
+    public User getUserById(@PathVariable String userId) throws ResponseStatusException {
```

Comment:
Dont return the whole user including password and other private data like email-adress